.release-it.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
We also do this in package.json, but it's fine here as well, I don't have a strong opinion. Both approaches has its pros and cons.